yargs/index.d.ts

```diff
@@ -187,6 +196,7 @@ declare namespace yargs {
         default?: any;
         defaultDescription?: string;
         demand?: boolean | string;
```

Comment:
You're right, the documentation still uses it in an example but not the list of valid keys.  I think the documentation is inconsistent.  I made an issue for it: https://github.com/yargs/yargs/issues/771